### ISSUE TITLE
twisted.spread: implements -> implementer

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7621]
|<img alt="wolfgang61's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| wolfgang61 reported|
|-|-|
|Trac ID|trac#7621|
|Type|enhancement|
|Created|2014-09-02 08:12:25Z|
|Branch|https://github.com/twisted/twisted/tree/pb-implementer-7621|

preparing for python3

Attachments:

* [spreadimplpatch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a49/a499c948ccc5f9b5108c9bd03c5ec8aecdc79f4b/75b48abc7db8b1d51e31e590c1c8d41619161577) (4973 bytes) - added by wolfgang61 on 2014-09-02 08:12:46Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7621 7621
type__enhancement enhancement
reporter__wolfgang61 wolfgang61
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_pb_implementer_7621 branches/pb-implementer-7621
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__py3k py3k
time__1409645545518812 1409645545518812
changetime__1410418668453245 1410418668453245
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

Patch looks reasonable to me.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

(In [43024]) Branching to pb-implementer-7621.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

(In [43025]) apply ‘spreadimplpatch’ re [#7621](https://github.com/twisted/twisted/issues/7621)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @glyph|
|-|-|

Build looks good.  Thanks for your contribution!  I'll merge it.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set status to `closed`|
|-|-|

(In [43026]) Merge pb-implementer-7621: use implementer() in twisted.spread

Author: wolfgang61

Reviewer: glyph

Fixes: [#7621](https://github.com/twisted/twisted/issues/7621)

Replace the use of zope.interface.implements with zope.interface.implementer in twisted.spread.
